<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet"  href="bootstrap-3.3.6-dist/css/bootstrap.min.css">
	<link rel="stylesheet"  href="bootstrap-3.3.6-dist/css/bootstrap-theme.min.css">
	<link rel="stylesheet"  href="bootswatch/simplex/bootstrap.min.css">
	<link rel="stylesheet" type="text/css" href="mystyle.css">
	<title>Phylogenetics Trees</title>

</head>
<body>
	<nav class="navbar navbar-inverse">
		<div class="container-fluid">
			<header class="navbar-brand">Varsha Koushik</header>
		</div>
		<!--<img src="" alt="image">!-->
		
	</nav>
	<div class="container">
		<ol class="breadcrumb">
			<li><a href="index.html">Projects</a></li>
			<li class="active">Phylogenetic Trees</li>
		</ol>
		<div class="row">
			<div class="col-xs-10 col-offset-2 col-sm-6">
				<h3>Reconstruction of Phylogenetic Trees</h3>
			</div>
			
		</div>
		<br>
		<br>
		<div class="row">
			<div class="col-xs-6">
				<img src="tree_Parsimony_sequential.png" width="100%" height="250px" alt="parsimony sequential tree">
				<figcaption align="center">Parsimony - Sequential Tree</figcaption>
			</div>
			<div class="col-xs-6">
				<img src="tree_Parsimony_parallel.png" width="100%" height="250px" alt="parsimony parallel tree">
				<figcaption align="center">Parsimony - Parallel Tree</figcaption>
			</div>
			
		</div>
		<br>
		<div class="row">
			<div class="col-xs-12">
				<p>Phylogenetics is the study of evolutionary relationships among various groups of organisms. Biologists use a multilevel taxonomy to classify organisms, giving categories at each level in the taxonomy. Where taxonomy classifies species based on their physical characteristics, habitats, and other such external features, phylogeny classifies species based on their evolutionary ancestry. As biologists have seldom been able to observe the evolutionary history, they infer from it instead. Inferences can be drawn in a phylogenetic tree. The project uses two of the many methods - UPGMA and Maximum Parsimony.
 				UPGMA (Unweighted Pair Group Method with Arithmetic Mean) is a distance based method which uses dissimilarities between sequences of various species to compute a distance matrix. This distance matrix aids in estimating how closely two species are related and thereby coming up with a phylogenetic tree. This algorithm makes an assumption that the rate of evolution of two species from a common ancestor is constant. The second method is Maximum Parsimony. Maximum parsimony predicts the evolutionary tree or trees that minimize the number of steps required to generate the observed variation in the sequences from common ancestral sequences. Thus evolutionary relationships can be studied and various inferences can be drawn about common ancestry.

				</p>
				<p>
					In this project, phylogentic trees were constructed using UPGMA and Maximum Parsimony with gene sequences as input.Parsimony was paralleized and compared for performance with its sequential counterpart.
				</p>
			</div>
		</div>
	</div>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
	<script src="/Users/varshakoushik/bootstrap-3.3.6-dist/js"></script>
</body>
</html>